docs/topics/auth/customizing.txt

```diff
@@ -407,7 +407,13 @@ use as your User model.
    have a ForeignKey to each other and seeing how ``makemigrations`` resolves that
    circular dependency if you want to see how it's usually done)
 
+.. warning::
 
+   Reusable apps don't use this feature. Remember: One project has N app. If 
+   app `fooapp` needs a custom `AUTH_USER_MODEL` `FooUser` and app `barapp` needs
+   `BarUser` you can't install both apps at once.
+   If you want to store per user information in your app, use a OneToOneField or inheritance.
```

Comment:
I am not sure about the suggestion to use inheritance. I guess that would be for adding custom methods, not storing per user information -- is that correct?